### DISCUSSION TITLE
Questions on correlation remove

### DISCUSSION THREAD

**Discussion by xingyumarkzhang1** [fairlearn/fairlearn#1121] [General]
I found the ‘Fairlearn’ open-source tools very helpful for me to learn the fairness method.
I have a question after reading the fairlearn preprocessing method.  I found that the only pre-processing mitigation method you applied is the correlation remover. However, I didn’t find any literature on this method. Do you mind me asking why did you choose this method? Is there any publication to document if this method works well in mitigating bias? Thanks

**Comment by hildeweerts** [comment]
Hi, @xingyumarkzhang1! Im so sorry for my late reply. I'm glad to hear you find fairlearn helpful!

The reason CorrelationRemover is included in fairlearn is mostly related to availability and simplicity of the approach. Basically, the folks from scikit-lego had already implemented the approach in their own library and proposed to implement it in fairlearn. There is no publication associated with the method, but the next release will contain more extensive documentation that might be helpful: https://fairlearn.org/main/user_guide/mitigation.html#correlation-remover

I hope this clears things up a bit. Let me know if you have any more questions!
